drivers/network/tcpip/tcpip/iinfo.c

```diff
@@ -72,7 +72,14 @@ TDI_STATUS InfoTdiQueryGetInterfaceMIB(TDIEntityID ID,
                     case NdisPhysicalMediumUnspecified:
 #endif
                     case NdisPhysicalMediumWirelessLan:
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
                         OutData->if_type = IF_TYPE_IEEE80211;
+#else
+                        /* Vista+ - IF_TYPE_IEEE80211,
+                         * 2003 - MIB_IF_TYPE_ETHERNET.
+                         */
```

Comment:
To me, this comment is redundant with simply reading the code.
If you add 1 comment for the 2 cases, then move it above the `#if`. And you can make it a one-liner.

Reply:
Done, thanks. Split the comment to a single line and moved it above.